src/Microsoft.ML.DataView/DataViewRowCursor.md

```diff
@@ -1,28 +1,28 @@
-﻿# `ICursor` Notes
+﻿# `DataViewRowCursor` Notes
 
 This document includes some more in depth notes on some expert topics for
-`ICursor` implementations.
+`DataViewRow` and `DataViewRowCursor` derived classes.
 
 ## `Batch`
 
-Some cursorable implementations, like `IDataView`, can through
-`GetRowCursorSet` return a set of parallel cursors that partition the sequence
-of rows as would have normally been returned through a plain old
-`GetRowCursor`, just sharded into multiple cursors. These cursors can be
-accessed across multiple threads to enable parallel evaluation of a data
-pipeline. This is key for the data pipeline performance.
+`IDataView.GetRowCursorSet` can return a set of parallel cursors that
+partition the sequence of rows as would have normally been returned through a
+more typical call to `GetRowCursor`. These cursors can be accessed across
+multiple threads to enable parallel evaluation of a data pipeline. This is key
+for the data pipeline performance during featurization.
 
 However, even though the data pipeline can perform this parallel evaluation,
 at the end of this parallelization we usually ultimately want to recombine the
 separate thread's streams back into a single stream. This is accomplished
-through `Batch`.
+through `DataViewRow.Batch`.
 
 So, to review what actually happens in ML.NET code: multiple cursors are
 returned through a method like `IDataView.GetRowCursorSet`. Operations can
```

Comment:
I would have started with this paragraph, because it is more descriptive. Feel like the beginning of the document lacks context.

Reply:
You mean the "so, to review" paragraph? I'm not so sure. I sort of feel like I am laying the necessary motivation up front, *then* hitting people with what it is -- now, necessarily, that will be a more "englightening" paragraph, but only because I think I spent some time laying out the logical groundwork for it.

But, I suppose, what I can do is just start with a small, mini-paragraph, saying that this is for this reonciliation purpose, so that this context in the broadest sense is at least established, since you're right, I do sort of begin leading people down a path without telling them in even the broadest sense where I am leading them...